Communitheme/gtk-3.0/_drawing.scss

```diff
@@ -61,11 +61,11 @@
     color: if($tc != $fg_color, transparentize($tc, .1), $fg_color);
     border-color: if($c != $base_color, transparent, $borders_color);
     background-color: $_bc;
-    @include _shadows(entry_focus_shadow(transparentize($fc, 1)), $_entry_edge);
+    @include _shadows(/*entry_focus_shadow(transparentize($fc, 1)),*/ $_entry_edge);
```

Comment:
It is not a good practice to comment out code. Just remove it! Trust the version control system.